cluster/up.sh

```diff
@@ -20,3 +20,28 @@ set -e
 source hack/common.sh
 source cluster/$MACPOOL_PROVIDER/provider.sh
 up
+
+# deploy kubevirt
+./cluster/kubectl.sh apply -f cluster/kubevirt/kubevirt-operator.yaml
```

Comment:
What about installing it directly from kubevirt release? I see no reason to keep manifests here. https://github.com/kubevirt/kubevirt/releases